src/components/donationItemForm/index.tsx

```diff
@@ -40,6 +40,23 @@ export default function DonationItemForm({
   onChange,
   disabled,
 }: DonationItemFormProps) {
+  // Tracking the current high/low values to show it instantly
+  const [highLowVals, setHighLowVals] = useState<[string, string]>([
+    'High',
+    'Low',
+  ]);
+
+  // Checking undefined, null, and emptiness before show the Higg/Low values
+  const updateHighLowVals = (newOrUsed: string, itemRes: ItemResponse) => {
+    if (!!newOrUsed) {
+      if (newOrUsed === 'Used') {
+        if (!!itemRes) {
+          setHighLowVals([`High ($${itemRes.high})`, `Low ($${itemRes.low})`]);
+        }
+      }
+    }
+  };
```

Comment:
```suggestion
  // Checking undefined, null, and emptiness before show the High/Low values
  const updateHighLowVals = (newOrUsed: string, itemRes: ItemResponse) => {
    if (newOrUsed === 'Used' && !!itemRes) {
      setHighLowVals([`High ($${itemRes.high})`, `Low ($${itemRes.low})`]);
    }
  };
```

Reply:
And correct me if I'm wrong, but `itemRes` is not nullable, so we should _always_ have a value there, yes? May be able to remove `!!itemRes` as well. Typescript shouuuuld ensure it. Double check me though.

Reply:
You are correct, I removed all the undefined/null evaluation and it still works fine.